RIAssigner/data/PandasData.py

```diff
@@ -67,6 +67,20 @@ def _sort_by_rt(self):
         """ Sort peaks by their retention times. """
         self._data.sort_values(by=self._rt_index, axis=0, inplace=True)
 
+    def __eq__(self, o: object) -> bool:
+        if not isinstance(o, PandasData):
+            return False
+        other: PandasData = o
+
+        are_equal = (self.retention_times == other.retention_times).all()
+        try:
+            are_equal &= (self.retention_indices == other.retention_indices).all()
+        except KeyError:
+            pass
+        are_equal &= self.filename == other.filename
```

Comment:
Are we sure the filename should be part of the comparison? imho filename shouldn't be trusted with data..

Reply:
I agree! Didn't think about it that way! Thank you for the comment!

Reply:
See https://github.com/RECETOX/RIAssigner/pull/51/commits/0a7450d6fd9e2557bde5f55279f0b8877cd3fc82